_static/js/custom.js

```diff
@@ -237,4 +237,13 @@ $(document).ready(() => {
   document.head.appendChild(instantPageScript);
+ 
+  let currentState = history.state;
+  // Remove domain and 'highlight' tag from URL.
+  let newURL = window.location.href.replace("https://docs.godotengine.org", "");
+  const reg = new RegExp('[?]highlight=([^?]*)');
+  newURL = newURL.replace(reg, "");
+
+  // Replace URL without reloading page.
+  history.replaceState(currentState, "", newURL);
```

Comment:
You might want to use the proper API instead of trying to come up with a regex that works (this one doesn't for instance, it breaks if there's a second param after this one) :wink:

```suggestion
  // Drop 'highlight' param from the URL.
  let url = new URL(location.href);
  url.searchParams.delete('highlight');

  // Replace URL without reloading page.
  history.replaceState(history.state, '', url);
```